src/pages/Homepage/Homepage.vue

```diff
@@ -74,34 +81,51 @@ export default {
 		query: activePageQuery,
 		preFetch: true,
 		result({ data }) {
-			// determine if take-over setting is active
-			const contentSetting = _get(data, 'contentful.entries.items', []).find(item => item.fields.key === 'ui-homepage-takeover'); // eslint-disable-line max-len
-			if (_get(contentSetting, 'fields')) {
-				const isContentEnabled = settingEnabled(
-					contentSetting.fields,
-					'active',
-					'startDate',
-					'endDate'
+			// lend-by-category as homepage experiment - EXP-GROW-138-Jul2020
+			const lendByCategoryExperiment = this.apollo.readFragment({
```

Comment:
@mlibazisi @ryan-ludwig @eddieferrer 

I just noticed that this probably needs a prior step to work. Looking into a couple of other experiment setups that were not initializing properly I found they were based solely on the settings and had not been assigned yet.

While this approach will work if you are testing via the force url it will not work if you are just relying on the setting distribution to initialize the experiment.

You'll need to create a prefetch function that chains the experiment assignment after the initial page query. See https://github.com/kiva/ui/pull/1920 for how that should be setup.

Reply:
@mcstover Thanks for the feedback. Will do this as a new PR